coursebook/week-6/README.md

```diff
@@ -34,16 +38,21 @@ and work through in pairs if time remaining (help each other first)!
 12:45pm - 1:45pm --
 [Database Testing](https://github.com/foundersandcoders/ws-database-testing/)
 
-2:45pm - 4:00pm -- [Introduce Project](./project.md) --
-[Research Afternoon](./research-afternoon.md) -- Review and Present research
-topics
+1:45pm - 2:00pm -- [Introduce Project](./project.md)
+
+2:00pm - 4:00pm -- [Research Afternoon](./research-afternoon.md) -- Review and
+Present research topics
 
 #### Day 3
 
 8:30am - 9:30am:
 [morning challenge](https://github.com/foundersandcoders/db-morning-challenge)
 
-9:30am onwards: projects
+9:30am - 10:30am Design Burst
```

Comment:
I think leave this in for now, but maybe this won't happen @jema28 ?

Reply:
Yeh, it definitely won't take  1 hour, 20 - 30 min tops. @ghassanmas It'll be a design resources session not a design burst 🐢

Reply:
I changed this to 9:30-10:00, I think we can merge now?